### ISSUE TITLE
management from web dashboard

### ISSUE THREAD

**Issue by VeRychard** [wyvern800/tldkp#6]
I've just found the awesome dashboard on the website. 

It would be great to have the management options available from there for convenience. Like adding/removing DKP for one person or all, or selected members. (connected to the other features, maybe the decay time when the cron runs, and latest bids/auctions)

Not super important but a QoL since we already have a nice UI which could be used for it

**Comment by wyvern800** [comment]
Will think about some things, because the main objective was to have everything set up by bot, having those on website makes things a bit redundant having the features in both sides. So I may think and see which ones are more important having done by website.

**Comment by VeRychard** [comment]
right. my idea came after seeing that the holding list is on the web. :)
regarding usability I agree it's better to keep stuff at one place

**Comment by ShioIna** [comment]
> Will think about some things, because the main objective was to have everything set up by bot, having those on website makes things a bit redundant having the features in both sides. So I may think and see which ones are more important having done by website.

I thought so as well. But I have to admit that working with the bot during bigger events is a bit cumbersome. Since you don't normally want your Guild members to be the ones to edit their own DKP values it would have to be done by someone from the officers.

I love the bot for adjusting DKP values for single Members on the fly, but I think it would be nice to have a feature that allows for batch editing of values.

**Comment by wyvern800** [comment]
Yes, you're right, I better implement a command for batch editing, and/or a way to be able to be editable on the website.
